pkg/build/tmpfolder.go

```diff
@@ -100,6 +99,18 @@ func (b *Build) mvProjectsToTmp() error {
 		b.cpLegacyProject()
 	} else if b.IsMod == true { // go 1.11, 1.12 has no Build.Root
 		b.cpGoModulesProject()
+		updated, newGoModContent, err := b.updateGoModFile()
+		if err != nil {
+			return fmt.Errorf("fail to generate new go.mod: %v", err)
+		}
+		log.Infof("go.mod needs rewrite? %v", updated)
```

Comment:
it's better to move this statment into the if scope as below, so that no log output if no need to update, and print detailed info otherwise.

Reply:
resolved.